### DISCUSSION TITLE
Implement new unsupported devices: Protocol Sniffing

### DISCUSSION THREAD

**Discussion by binoz** [rytilahti/python-miio#1422] [Q&A]
Hi, i would like to implement support for Xiaomi Mop 2S, but i'm having some problems figuring out how to setup Wireshark and/or Android emulator for sniffing the protocol. I tried using bluestack with Xiaomi Home installed + Wireshark but i was not able to sniff any traffic. Any one willing to share his own experience?

**Comment by rytilahti** [comment]
Could it be that this specific model (or manufacturer) doesn't use local communications at all but everything gets delivered over the cloud? Are you able to run `miiocli info` against the device locally?
